packages/appstore/src/components/elements/text/balance-text.tsx

```diff
@@ -16,16 +16,15 @@ type BalanceTextProps = {
 };
 
 const BalanceText = ({ balance, currency, size = 'm', underline_style = 'none' }: BalanceTextProps) => {
-    const { client, traders_hub } = useStores();
+    const { traders_hub } = useStores();
     const { selected_account_type } = traders_hub;
-    const { has_active_real_account } = client;
 
     const getTextClassName = () => {
         if (selected_account_type === 'demo') {
-            return has_active_real_account ? 'balance-text__text--demo' : 'balance-text__text--amount';
+            return 'balance-text__text--demo';
```

Comment:
What is the reason of removing this condition ?

Reply:
that line of code gets the balance from the store and checks if the balance is 0.00
if the balance is 0.00, color is black, regardless if it's demo or real account

I changed the code so that the color is green/blue depending on demo/real account, so if the balance is 0.00 it won't become black (referring to provided figma)

Reply:
![image](https://user-images.githubusercontent.com/108507236/222332243-2b039652-3582-497e-9a38-56c491fb42fe.png)